src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -46,7 +46,7 @@ Note you need to register the [tooltip directive](https://nextcloud-vue-componen
 			:maxlength="400"
 			:multiline="true"
 			:user-data="userData"
-			placeholder="Try mentioning the user Test01"
+			placeholder="Try mentioning user @Test01 or inserting emoji :smile"
```

Comment:
```suggestion
			placeholder="Try mentioning user @Test01 or inserting emoji :smile:"
```
Typo?

Reply:
I decided not to append a colon on purpose. The trailing colon is not needed for autocompletion and makes it harder to read in my opinion. It's more that the prepending colon is the trigger for autocompletion, and then you type your search query (e.g. "smile").

I agree that this might change when we implement automatic conversion of `:smile:` to an emoji, but I would say that's out of scope here :blush: